### ISSUE TITLE
[Bug]:  RAPPE tab accessibility testing

### ISSUE THREAD

**Issue by yamenAl** [fdnd-agency/footguard#233]
During tab accessibility testing, I found that keyboard navigation using the Tab key does not work correctly in Safari.

**Where is did not work.**
Hamburger menu
Gradings page

**Behavior:** Tab navigation does not respond 

<hr>

**Screen shot:**

**Humburger menu:**
<img width="200" height="1000" alt="Image" src="https://github.com/user-attachments/assets/4d068d41-018a-4d1f-a055-460c1747a8aa" />

<hr>

**Grading**
<img width="1700" height="1016" alt="Image" src="https://github.com/user-attachments/assets/c847f012-6b63-4b94-b3e7-3ea61fdc72d9" />
